docker-compose.yml

```diff
@@ -47,6 +47,11 @@ services:
       - OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
       - OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE
       - OTEL_SERVICE_NAME=adservice
+      - OTEL_TRACES_EXPORTER=otlp
```

Comment:
These have to be here for the current version of our OTEL agent. We could instead keep the defaults and document how to turn off exporters to not see errors when not using them.

Here's where the defaults are changed: https://github.com/getsentry/sentry-java/blob/7967d22fda22c5c425b71fdfd67b15f5419db792/sentry-opentelemetry/sentry-opentelemetry-agentcustomization/src/main/java/io/sentry/opentelemetry/SentryAutoConfigurationCustomizerProvider.java#L95-L96